channels/uk.m3u

```diff
@@ -439,7 +439,7 @@ https://tennischannel-intl-samsung-uk.amagi.tv/playlist.m3u8
 https://timeline-samsung-uk.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="TJC.uk" tvg-name="TJC" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/cFlNhjV.png" group-title="",TJC (1080p)
 https://cdn-shop-lc-01.akamaized.net/Content/HLS_HLS/Live/channel(TJCOTT)/index.m3u8
-#EXTINF:-1 tvg-id="Toongoggles.uk" tvg-name="Toongoggles" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/8PEf3Vn.jpg" group-title="",Toongoggles (720p)
+#EXTINF:-1 tvg-id="Toongoggles.uk" tvg-name="Toongoggles" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/8PEf3Vn.jpg" group-title="Animation",Toongoggles (720p)
```

Comment:
Not only animation. Family Friendly programming for sure, but not only animation. Right now there's a dancing program being aired.